cluster/charts/crossplane/README.md

```diff
@@ -78,6 +78,7 @@ and their default values.
 | `leaderElection` | Enable leader election for Crossplane Managers pod | `true` |
 | `nodeSelector` | Enable nodeSelector for Crossplane pod | `{}` |
 | `customLabels` | Custom labels to add into metadata | `{}` |
+| `serviceaccount.customAnnotations` | Custom Annotations to add into metadata of sercviceaccount | `{}` |
```

Comment:
Would be good to specify that these are only added to the Crossplane `ServiceAccount`.